analytics/articles.js

```diff
@@ -72,6 +72,14 @@ if(location.pathname.match('/article/')){
     });
   });
 
+  analyticsHooks.on('impressions:editorial-signup', function(options){
+    analytics.track('Viewed popup for editorial email', {
+      article_id: options.articleId,
+      context_type: options.type,
```

Comment:
Based on what you have now, the `context_type = "article_fixed"`, based on the `getType()` function, but we would probably want the type to be `article_popup` instead. This [doc](https://docs.google.com/spreadsheets/d/17oqFli15x1SmNBu_-fPZhFifMw2BNSPJe9VdOgs1aKs/edit#gid=1499052602) is what we use to stay aligned with the Analytics team on naming. Sorry I hadn't shared this with you before! 

```
context_type: 'article_popup'
impression_type: 'newsletter_signup'
```